packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-title-icon.tsx

```diff
@@ -8,11 +8,14 @@ import { CFD_PLATFORMS, MARKET_TYPE_SHORTCODE } from '../../Helpers/cfd-config';
 
 const CFDCompareAccountsTitleIcon = ({ trading_platforms, is_eu_user, is_demo }: TCompareAccountsCard) => {
     const market_type = !is_eu_user ? getMarketType(trading_platforms) : 'CFDs';
-    const market_type_shortcode = market_type.concat('_', trading_platforms.shortcode ?? '');
+    const market_type_shortcode =
+        trading_platforms.platform === CFD_PLATFORMS.MT5 && market_type === 'all'
```

Comment:
market_type === 'all' is already defined in the config file.. let's not hardcode it.